webapp/app/components/pages/login/LoginPage.css

```diff
@@ -0,0 +1,27 @@
+.card-container.card {
+    max-width: 350px;
+    padding: 40px 40px;
+}
+
+.card {
+    background-color: #F7F7F7;
+    padding: 20px 25px 30px;
+    margin: 0 auto 25px;
+    margin-top: 50px;
```

Comment:
no need to override margin-top separately, it could be defined directly like so

`margin: [top] [horizontal] [bottom];`

Please replace the following
```
margin: 0 auto 25px;
margin-top: 50px;
```
with the appropriate definition
`margin: 50px auto 25px;`